.github/workflows/winget.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish on Windows Package Manager
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: Release tag (version)
+        required: true
+
+jobs:
+  publish-winget:
+    name: Publish
+    runs-on: windows-latest
+    if: github.event_name == 'workflow_dispatch' || github.event_name == 'release' && !github.event.release.prerelease
+    steps:
+      - name: Configure parameters
+        id: params
+        run: |
+          if ("${{ github.event_name }}" -eq "release") {
+            # Sometimes assets aren't immediately available, so we wait a bit for the first run
+            ${{ github.run_attempt == 1 && 'Start-Sleep -Seconds 300' || '' }}
```

Comment:
What would trigger a second run attempt of the workflow here?

Reply:
In the case that something failed and you wanna run it manually.